projects/custody-key-dates-and-delius/src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/IntegrationTest.kt

```diff
@@ -83,6 +83,44 @@ internal class IntegrationTest {
         )
     }
 
+    @Test
+    fun `Custody Key Dates updated from SENTENCE_CHANGED event`() {
+        val notification = Notification(
+            message = MessageGenerator.SENTENCE_CHANGED,
+            attributes = MessageAttributes(eventType = "SENTENCE_CHANGED")
+        )
+
+        val first = CompletableFuture.runAsync {
+            channelManager.getChannel(queueName).publishAndWait(notification, Duration.ofMinutes(3))
```

Comment:
Do we still need the 3 minute wait here?

Reply:
removed